lib/Component/VCard.php

```diff
@@ -83,7 +84,7 @@ class VCard extends VObject\Document {
         'ORG'     => 'Sabre\\VObject\\Property\\Text',
         'NOTE'    => 'Sabre\\VObject\\Property\\FlatText',
         'REV'     => 'Sabre\\VObject\\Property\\VCard\\TimeStamp',
-        'SOUND'   => 'Sabre\\VObject\\Property\\FlatText',
+        'SOUND'   => 'Sabre\\VObject\\Property\\Uri',
```

Comment:
We need to make sure this will behave well for vCard 3 also. In cases where the default value type changed between 3 and 4, I went for what the vCard 3 spec had to say about it, as this was more popular.

Ideally we alternate the default value type depending on the vcard version... but if we don't do this right now, we should go back to vCard 3 defaults.